static/src/components/builder/store/modules/multiselectInput.js

```diff
@@ -0,0 +1,93 @@
+import * as types from '../types';
+import utils from '../../utils';
+
+export function initialState () {
+  const firstElement = getChoiceObject();
+  return {
+    label: '',
+    labelAdded: false,
+    choiceList: [firstElement],
+    pybAnswer: '',
+    initialValue: '',
+    validations: []
+  };
+}
+
+export function getChoiceObject () {
+  return {
+    display: '',
+    value: ''
+  };
+}
+
+export const state = {
+  ...initialState()
+};
+
+function* getErrors (state) {
+    const uniqueValues = new Set();
+    for (const [index, { value }] of state.choiceList.entries()) {
+        const key = `choiceList[${index}].value`;
+        const trimmedValue = value.trim();
+        if (!trimmedValue) yield [key, `Choice ${index + 1} value is required.`];
+        else if (uniqueValues.has(trimmedValue)) yield [key, `Choice ${index + 1} value is not unique.`];
+        else uniqueValues.add(trimmedValue);
+    }
+}
+
+export const getters = {
+  [types.GET_MULTISELECT_INPUT_PROPS] (state) {
+    return {
+      label: state.label,
+      labelAdded: state.labelAdded,
+      choices: state.choiceList.map((choice) => { return choice.value.trim(); }),
```

Comment:
For ES6 arrow function, curly braces and 'return' keyword are redundant for single return statement. The bracket for single parameter can also be omitted. If you like, you can write this line as choices: state.choiceList.map(choice => choice.value.trim())
This comment applies to multiple places in the pr :)

Reply:
Ok, thanks Xi! I will fix these.